.github/actions/generate-mapping/action.yml

```
@@ -0,0 +1,25 @@
+---
+name: 'Generate User Cache Mapping'
+description: 'Creates json file of slack users by github or member login'
+author: 'Fieldguide'
+inputs:
+  github_token:
+    description: 'Repository GITHUB_TOKEN or personal access token secret'
+    required: false
+    default: ${{ github.token }}
+  org:
```

Could we prefix this?

```suggestion
  github_org:
```